packages/testing/src/testing.ts

```diff
@@ -93,12 +93,11 @@ export function createTestingPinia({
 
   setActivePinia(pinia)
 
-  return Object.assign(
-    {
-      get app(): App {
-        return (this as TestingPinia)._a
-      },
+  Object.defineProperty(pinia, 'app', {
+    get: function (): App {
```

Comment:
Let's keep it like before:

```suggestion
    configurable: true,
    enumerable: true,
    get(): App {
```